app/agents/voice/automatic/__init__.py

```
@@ -117,8 +108,123 @@ async def main():
+
+        # Wait for session assignments
+        while True:
+            try:
+                line = sys.stdin.readline()
+                if line.strip():
+                    session_config = json.loads(line.strip())
+                    await handle_session(session_config)
+            except EOFError:
+                logger.info("Pool process received EOF, shutting down")
+                break
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)